src/stavisky_lab_to_nwb/redis_interfaces/redisrecordinginterface.py

```diff
@@ -0,0 +1,54 @@
+"""General recording interface for Redis stream data."""
+import redis
+import numpy as np
+from pynwb.file import NWBFile
+from typing import Union, Optional, List, Tuple, Literal
+
+from neuroconv.datainterfaces.ecephys.baserecordingextractorinterface import BaseRecordingExtractorInterface
+
+from stavisky_lab_to_nwb.redis_interfaces.redisrecordingextractor import RedisStreamRecordingExtractor
+
+
+class RedisStreamRecordingInterface(BaseRecordingExtractorInterface):
+    """Recording interface for Redis stream data"""
+
+    ExtractorModuleName = "stavisky_lab_to_nwb.redis_interfaces.redisrecordingextractor"
+    ExtractorName = "RedisStreamRecordingExtractor"
+
+    def __init__(
+        self,
+        port: int,
+        host: str,
+        stream_name: str,
+        data_key: str,
+        dtype: Union[str, type, np.dtype],
+        channel_count: int,
+        channel_ids: Optional[list] = None,
+        frames_per_entry: int = 1,
+        start_time: Optional[float] = None,
```

Comment:
I see why `start_time` is a part of the `__init__` for the SpikeInterface extractor, but the NeuroConv API would prefer this be handled automatically, probably like how you have it setup in the conversion script